### PR TITLE
update project config & guide doc

### PR DESCRIPTION
**Pull request by fanqiang001**
1.Modify the default project configuration options, wince with SDL dynamic library.
2.Update《各平台编译发布指南》doc , increase the QT SDK for wince compile way.

### CODE REVIEW

File: Gen3UI/Gen3UI.pro [hunk 1]
```diff
@@ -12,7 +12,7 @@ TARGET = Gen3UI
 TEMPLATE = lib
 
 CONFIG += ffmpeg
-#CONFIG  += wince  ##wince
+CONFIG  += wince  ##wince
```
Comment (BrandonHe) [inline Gen3UI/Gen3UI.pro:15]: I am not sure set the WinCE port as default port is necessary, maybe mentioned this in the instruction doc is better.

File: Gen3UI/Library/ce/sdl/smartDeviceLink.ini [hunk 2]
```diff
@@ -31,7 +31,7 @@ AudioStreamingPort = 5080
 ; represents as a git commit hash value
 SDLVersion = {GIT_COMMIT}
 ; All logging event could be dropped by setting $LogsEnabled to false
-LogsEnabled = false
+LogsEnabled = true
```
Comment (BrandonHe) [inline Gen3UI/Library/ce/sdl/smartDeviceLink.ini]: Should set `LogsEnable` default value to true?

File: hmi_sdk/hmi_sdk.pro [hunk 3]
```diff
@@ -11,7 +11,7 @@ win32:QT += axcontainer
 #DEFINES +=VIDEO_STREAM_WIDGET
 
 DEFINES += __STDINT_MACROS  #for ffmpeg
-#CONFIG  += wince  ##wince
+CONFIG  += wince  ##wince
```
Comment (BrandonHe) [inline hmi_sdk/hmi_sdk.pro:14]: So, now the default HMI sdk port is for WinCE?

**Comment by fanqiang001** [comment]
1.Modify smartDeviceLink.ini default value.
2.Wince platform videostream default mode of communication is callback.

**Comment by BrandonHe** [comment]
@fanqiang001 
Have you tested with new changes?

**Comment by fanqiang001** [comment]
yes, i had.

File: hmi_sdk/hmi_sdk.pro (hunk 3 above)
Comment (fanqiang001) [reply hmi_sdk/hmi_sdk.pro:14]: yes

**Comment by BrandonHe** [comment]
Reviewed & Approved

**Pull request merged**